### PR TITLE
added formula for jxcore

### PR DESCRIPTION
**Pull request by nexus-uw**

### CODE REVIEW

**Comment by DomT4** [comment]
Some problems:

* The tool is currently in beta - ` JXcore Beta 2.3.7 `.
* You're not actually configuring anything because you're using the binary download. There doesn't seem to be a source download available at this point. The zip you're using only contains ` jx `, the ` LICENSE ` and the release notes.
* The binary being used only supports 64 bit Intel machines, so you'd need to exclude all the PPC arches.

**Comment by mikemcquaid** [comment]
@BrewTestBot test this please

File: Library/Formula/jx_osx.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  test do
+    system "false"
```
Comment (mikemcquaid) [inline Library/Formula/jx_osx.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/jx_osx.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class JxOsx < Formula
+  homepage "http://jxcore.com/"
+  url "https://s3.amazonaws.com/nodejx/jx_osx64.zip"
```
Comment (mikemcquaid) [inline Library/Formula/jx_osx.rb]: This looks like a binary. If it is one it should be submitted to `homebrew-binary` instead.

**Pull request closed without merging**